src/btwa_api/app/config.py

```diff
@@ -8,12 +8,19 @@
 from pyhocon import ConfigFactory
 
 ROOT_DIR = pathlib.Path(__file__).absolute().parent.parent
+
+# SQL database
```

Comment:
do we still need the SQL db? if not, get rid of everything related to it and keep the Firebase-related stuff only

Reply:
We use it to authenticate the backend users (epidemiologists). If we switch to a different login mechanism, we can ditch it (but we will probably need some small DB for tokens and such).

Reply:
Got you